openPrompt/1st_to_3rd_person_writing/prompts.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
better to add the urls of the source OpenAI Community and personal blogs in the commit message body